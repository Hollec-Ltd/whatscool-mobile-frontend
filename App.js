import React from 'react';
import AppNavigator from './src/navigations';

export default function App() {
  return <AppNavigator />;
}
