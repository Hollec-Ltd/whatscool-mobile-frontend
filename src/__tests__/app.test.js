import React from 'react';
import { render } from 'react-native-testing-library';
import App from '../../App';

describe('Entry file', () =>{
    it ('Should render app component properly',() => {
        expect(<App/>).toMatchSnapshot();
    })
})