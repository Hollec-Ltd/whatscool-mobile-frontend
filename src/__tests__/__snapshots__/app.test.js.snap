// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Entry file Should render app component properly 1`] = `<App />`;
